src/fragment-components/a-textinput.tsx

```diff
@@ -25,7 +25,7 @@ export const ATextInputStyleUI = ({selectedComponent, setComponent}: any) => {
 			onChange={(event: any) => setComponent({
 				...selectedComponent,
 				inputType: event.selectedItem.id
-		})}/>
+			})} />
```

Comment:
Check indentation locally.

Reply:
Same situations as the other posted images